common/council_positions/00_council_positions.txt

```diff
@@ -161,8 +161,9 @@ councillor_spymaster = {
 
 councillor_court_chaplain = {
 	skill = learning
-
-	auto_fill = yes
```

Comment:
Mmm, don't think it's a good idea to remove it.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/99992600-f9f51200-2dcf-11eb-97da-a498fc2b81e2.png)

</details>

Reply:
It still fills the position, and changing chaplains does not trigger the bug.